en/console-and-shells.rst

```diff
@@ -468,9 +469,15 @@ You can mark output as follows::
     $this->out('extra message', 1, Shell::VERBOSE);
     $this->verbose('Verbose output');
 
-You can control the output level of shells, by using the ``--quiet`` and ``--verbose``
-options. These options are added by default, and allow you to consistently control
-output levels inside your CakePHP shells.
+You can control the output level of shells, by using the ``--quiet`` and
+``--verbose`` options. These options are added by default, and allow you to
+consistently control output levels inside your CakePHP shells.
+
+The ``--quiet`` and ``--verbose`` options also control how logging data is
+output to stdout/stderr. Normally info and higher log messages are output to
```

Comment:
Yeah, it seems. It sounded right in my ears anyway :smirk_cat: 

> Preterite
I outputted
I output

http://conjugator.reverso.net/conjugation-english-verb-output.html